src/index.js

```diff
@@ -8,8 +8,6 @@ require('dotenv').config();
 
 const FIREHOSE_AUTH_KEY = 'x-firehose-key';
 
-program.option('--keep-alive');
-
 program.parse();
```

Comment:
**suggestion:** Let's lean into command line arguments and away form environment variables

It makes it easier to understand what the program needs (`--help`) versus reading docs/docs